pyne/dbgen/simple_xs.py

```diff
@@ -29,19 +29,19 @@ def grab_kaeri_simple_xs(build_dir=""):
     build_dir : str
         Major directory to place html files in. 'KAERI/' will be appended.
     """
-    zip_path = os.path.join(build_dir, "kaeri.zip")
-    zip_url = "http://data.pyne.io/kaeri.zip"
+    zip_path = os.path.join(build_dir, 'kaeri.zip')
+    zip_url = 'http://raw.githubusercontent.com/pyne/data/master/kaeri.zip'
```

Comment:
Can we use HTTPS for secure connection?
```suggestion
    zip_url = 'https://raw.githubusercontent.com/pyne/data/master/kaeri.zip'
```